mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -274,6 +274,29 @@ class Word2VecModel private[ml] (
     wordVectors.findSynonyms(word, num)
   }
 
+  /**
```

Comment:
Yes, so as I mentioned you could do the map function with the `_1()` and `_2()` to convert it entirely in the Python side.